data/manthanank.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "Manthan Ank",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**Front End** Developer | Angular | JavaScript | Nodejs | MongoDB | Open Source",
+    "avatar": "https://github.com/manthaank.png",
+    "tags": ["Open Source", "Javascript", "Typescript", "Angular", "Nodejs", "MongoDB"],
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/manthaank",
+        "icon": "FaGithub"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/manthan_ank",
+        "icon": "FaTwitter"
+      },
+      {
+        "name": "Check out my Portfolio",
+        "url": "https://manthanank.github.io/",
+        "icon": "FaLink"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/manthanank/",
+        "icon": "FaLinkedin"
+      }
+    ],
+    "milestones": [],
+    "testimonials": []
+  }
+  
```

There's an extra line here.
```suggestion
}
```